src/lib/i18n/locales/ar-BH/translation.json

```diff
@@ -0,0 +1,443 @@
+{
```

Comment:
translation.json should look something like this: https://github.com/open-webui/open-webui/blob/main/src/lib/i18n/locales/bg-BG/translation.json

Reply:
hahaha oh sorry my mistake now I understand

Reply:
fix Thank you for your help @tjbck